src/scripts/annotate.pl

```diff
@@ -464,10 +469,12 @@ sub is_bad_fusion{
   open(my $JUNC, "$junction_file");
   while(my $line = <$JUNC>){
 	chomp($line);
-	next unless($line =~ m/novel/ || $line =~ m/chrX:1331/);
+	# chrX:1212 is the hg38 coordinate prefix for CRLF2 (chrX:1,190,449-1,212,815(GRCh38/hg38))
+	# chrX:1331 is the hg19 coordinate prefix for CRLF2 (chrX:1,314,869-1,331,616(GRCh37/hg19))
+	next unless($line =~ m/novel/ || $line =~ m/chrX:1212/ || $line =~ m/chrX:1331/);
```

Comment:
Is there a way we can do this without hardcoding?

Reply:
My understanding is that there is some issue where CRLF2 fusions get missed sometimes. I don't know what causes it, but this was the "solution". I'm going to create an issue because I'll have to check with @liqingti to see what sample had this issue. Looking at this a little closer, it is changing the filter for junctions reads from RNApeg.

Reply:
Based on my understanding, this is specific for short deletion derived CRLF2 fusion (e.g. P2RY8-CRLF2 in the paper). We can only detect this kind of fusion through "novel junction" from RNApeg output.
However, sometimes RNApeg output does not put the junction as novel, that is the reason we need "next unless($line =~ m/novel/ || $line =~ m/chrX:1212/);"
For "unless($line =~ m/chrX:1212/)", it means the criteria is not for CRLF2. Because CICERO wants to detect CRLF2 fusion sensitively.

Reply:
@liqingti - I'm sure the motivation for the logic is consistent so fully support the idea here. My only concern is that hardcoding the locations in the code itself is not highly modular. We typically use reference files, etc. for this kind of thing instead of hardcoding in the code itself. I'm not sure if it is a better idea for us to use a new ref file here, hardcode, or do something else entirely. Just wanted to point out that this doesn't seem like a best practice for software development is all.